whipper/common/program.py

```diff
@@ -447,6 +456,9 @@ def getTagList(self, number, mbdiscid):
                 tags['MUSICBRAINZ_ALBUMID'] = mbidRelease
                 tags['MUSICBRAINZ_RELEASEGROUPID'] = mbidReleaseGroup
                 tags['MUSICBRAINZ_ALBUMARTISTID'] = mbidReleaseArtist
+                if classical:
```

Comment:
Do like the test for "MUSICBRAINZ_WORKID" instead here. Even if `classical` mode is enabled, there may not be any composer/performer data available—esp. with the current system of wholly relying on relationship data, which may not always be present… but even with my suggestion of making classical use Track/Recording Artist data, doing a "MUSICBRAINZ_WORKID" style check means we won’t have to do different checks depending on what mode the composer/performer data is fetched through.

Reply:
I'm not sure if my "MUSICBRAINZ_WORKID" style check works tho :-\ .